src/commands/sub/snowflake/compare.ts

```diff
@@ -0,0 +1,35 @@
+import type { ArgsParam, InteractionParam } from "@yuudachi/framework/types";
+import { SnowflakeUtil, time, TimestampStyles } from "discord.js";
+import type { SnowflakeCommands } from "../../../interactions/index.js";
+import { Colors } from "../../../util/constants.js";
+
+export async function compare(interaction: InteractionParam, args: ArgsParam<typeof SnowflakeCommands>["compare"]) {
+	if (!/^\d{17,20}$/gi.test(args.snowflake1) || !/^\d{17,20}$/gi.test(args.snowflake2)) {
+		await interaction.reply({
+			content: "Invalid snowflake",
+			ephemeral: true,
+		});
+		return;
+	}
+
+	await interaction.deferReply({ ephemeral: true });
+
+	const snowflake1 = SnowflakeUtil.deconstruct(args.snowflake1);
+	const snowflake2 = SnowflakeUtil.deconstruct(args.snowflake2);
```

Comment:
Why not reference the timestamp directly here and pass only the variable to the arguments:
- Timestamp is already a number, using deconstruction or the timestamp utility, no need for conversion

```suggestion
	const snowflake1 = SnowflakeUtil.timestampFrom(args.snowflake1);
	const snowflake2 = SnowflakeUtil.timestampFrom(args.snowflake2);
```